src/app/containers/ArticleTimestamp/timeFormatTests/expectedFormats.json

```diff
@@ -0,0 +1,42 @@
+{
+  "arabic":{
+     "exact date":"10 أكتوبر/ تشرين الأول 2018",
+     "three hours ago":"منذ 3 ساعات",
+     "five hours ago":"منذ 5 ساعات",
+     "eleven hours ago":"منذ 11 ساعة",
+     "twelve hours ago":"منذ 12 ساعة",
+     "twenty four hours ago":"منذ يوم واحد",
+     "two days ago":"منذ يومين",
+     "three days ago":"منذ 3 أيام"
+  },
+  "igbo":{
+     "exact date":"10 Ọktọba 2018",
+     "three hours ago":"elekere 3 gara aga",
+     "five hours ago":"elekere 5 gara aga",
+     "eleven hours ago":"elekere 11 gara aga",
+     "twelve hours ago":"elekere 12 gara aga",
+     "twenty four hours ago":"otu ụbọchị gara aga",
+     "two days ago":"Ụbọchị 2 gara aga",
+     "three days ago":"Ụbọchị 3 gara aga"
+  },
+  "news":{
+     "exact date":"10 October 2018",
+     "three hours ago":"3 hours ago",
+     "five hours ago":"5 hours ago",
+     "eleven hours ago":"11 hours ago",
+     "twelve hours ago":"12 hours ago",
+     "twenty four hours ago":"a day ago",
+     "two days ago":"2 days ago",
+     "three days ago":"3 days ago"
+  },
+  "persian":{
+     "exact date":"۱۸ مهر ۱۳۹۷ - ۱۰ اکتبر ۲۰۱۸",
+     "three hours ago":"۳ ساعت پیش",
+     "five hours ago":"۵ ساعت پیش",
+     "eleven hours ago":"۱۱ ساعت پیش",
+     "twelve hours ago":"۱۲ ساعت پیش",
+     "twenty four hours ago":"یک روز پیش",
+     "two days ago":"۲ روز پیش",
+     "three days ago":"۳ روز پیش"
+  }
+}
```

Comment:
Please can you share the script you used to generate this?

Also please could you add a space after the colons for consistency with our other JSON files?

Reply:
@jamesbrumpton I think we've got something like that on storybook for the` ArticleTimestamp` container, this pr only covers the logic for the timestamp formats across services

Reply:
Let's reinstate the part of the Cypress test that checsk the presence of the 'Updated' string (`articleTimestampPrefix`)

Reply:
@jamesbrumpton see above where this test has been added at https://github.com/bbc/simorgh/pull/3894/files#diff-7292dfc25cc01741765752817c268032R176-R179

Reply:
Do we need to check for a suffix as well or is that not part of our requirements?